Formula/abcl.rb

```diff
@@ -40,6 +40,6 @@ def install
 
   test do
     (testpath/"test.lisp").write "(print \"Homebrew\")\n(quit)"
-    assert_match /"Homebrew"$/, shell_output("#{bin}/abcl --load test.lisp").strip
+    assert_match(/"Homebrew"$/, shell_output("#{bin}/abcl --load test.lisp").strip)
```

Comment:
Doesn't need to be a regex

Reply:
So, after all of that, I realized that this example is still a regex...

The reason I didn't convert these is in case the reason the `$` was present was to prevent some specific match that would indicate a bug. Is it safe to assume that in cases where the only regex-thing in the literal is a `^` or a `$`, the regex should still be converted to string, dropping the regex characters?